arch/cortex-m4/src/scb.rs

```diff
@@ -33,6 +33,22 @@ const SCB_BASE: usize = 0xE000ED00;
 
 static mut SCB: *mut ScbRegisters = SCB_BASE as *mut ScbRegisters;
 
+/// Allow the core to go into deep sleep on WFI.
+///
+/// The specific definition of "deep sleep" is chip specific.
+pub unsafe fn set_sleepdeep() {
```

Comment:
I think this is valid. Here is a cortex-m4 implementation. Whether a chip is allowed to use it or not is chip-specific.